core/io/pck_packer.cpp

```diff
@@ -218,6 +237,10 @@ Error PCKPacker::flush(bool p_verbose) {
 
 	int count = 0;
 	for (int i = 0; i < files.size(); i++) {
+		if (files[i].ofs == 0 && files[i].size == 0 && files[i].src_path == String()) {
```

Comment:
I suppose this is what it's for?
```suggestion
		if (files[i].ofs == FILE_REMOVED_OFFSET && files[i].size == 0 && files[i].src_path == String()) {
```